src/scipp/html/style.css.template

```diff
@@ -235,6 +235,8 @@
   grid-column: 4;
   text-align: left;
   color: var(--sc-font-color1);
+  max-width: 80px;
```

Comment:
I think it would be better to use `pt` here and assume that the user has a font size that works for their screen. 80px could be very small in some cases.